server/routes/bulkMembers.ts

```diff
@@ -4,17 +4,49 @@ import { factory, formatFilename } from '../../shared/logging';
 import { DB } from '../database';
 
 const log = factory.getLogger(formatFilename(__filename));
-const bulkMembers = async (models: DB, req: Request, res: Response, next: NextFunction) => {
-  const [chain, community, error] = await lookupCommunityIsVisibleToUser(models, req.query, req.user);
+const bulkMembers = async (
+  models: DB,
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  const [chain, community, error] = await lookupCommunityIsVisibleToUser(
+    models,
+    req.query,
+    req.user
+  );
   if (error) return next(new Error(error));
 
+  const total = await models.Role.count({
+    where: chain
+      ? { chain_id: chain.id }
+      : { offchain_community_id: community.id },
+  });
+
   const members = await models.Role.findAll({
-    where: chain ? { chain_id: chain.id } : { offchain_community_id: community.id },
-    include: [ models.Address ],
-    order: [['created_at', 'DESC']],
+    where: chain
+      ? { chain_id: chain.id }
+      : { offchain_community_id: community.id },
+    include: [models.Address],
+    order: [
+      [
+        { model: models.Address, as: 'Address' },
+        'last_active',
+        'DESC NULLS LAST',
+      ],
+    ],
+    offset:
+      req.query.itemsPerPage && req.query.pageNum
+        ? req.query.itemsPerPage * req.query.pageNum
+        : undefined,
+    limit: req.query.itemsPerPage,
```

Comment:
What do these query changes do?

Reply:
These are from @gdjohnson's PR that fixed the pagination for offchain-community fetching.